.github/workflows/ci.yml

```
@@ -0,0 +1,60 @@
+name: CI
+
+on:
+  push:
+    # Avoid duplicate builds on PRs.
+    branches:
+      - main
+    tags: 
+      - v*
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  shellcheck:
+    runs-on: ubuntu-22.04
+    container:
+      image: koalaman/shellcheck-alpine:v0.8.0
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run shellcheck
+      run: find . -type f \( -name "*.sh" -o -path "*/bin/*" \) ! -name '*.jq' | xargs -t shellcheck
+  build-stack:
+    runs-on: ubuntu-22.04
+    needs:
+    - shellcheck
+    env:
+      STACK: heroku-${{ matrix.stack-version }}
+      STACK_VERSION: "${{ matrix.stack-version }}"
+      DOCKER_HUB_TOKEN: "${{ secrets.DOCKER_HUB_TOKEN }}"
+      ID_SERVICE_PASSWORD: "${{ secrets.ID_SERVICE_PASSWORD }}"
+      MANIFEST_APP_TOKEN: "${{ secrets.MANIFEST_APP_TOKEN }}"
```

Some env vars were missing:

```suggestion
      DOCKER_HUB_TOKEN: "${{ secrets.DOCKER_HUB_TOKEN }}"
      DOCKER_HUB_USERNAME: "${{ secrets.DOCKER_HUB_USERNAME }}"
      ID_SERVICE_PASSWORD: "${{ secrets.ID_SERVICE_PASSWORD }}"
      ID_SERVICE_TOKEN_ENDPOINT: "${{ secrets.ID_SERVICE_TOKEN_ENDPOINT }}"
      ID_SERVICE_USERNAME: "${{ secrets.ID_SERVICE_USERNAME }}"
      INTERNAL_REGISTRY_HOST: "${{ secrets.INTERNAL_REGISTRY_HOST }}"
      INTERNAL_REGISTRY_USERNAME: "${{ secrets.INTERNAL_REGISTRY_USERNAME }}"
      MANIFEST_APP_TOKEN: "${{ secrets.MANIFEST_APP_TOKEN }}"
      MANIFEST_APP_URL: "${{ secrets.MANIFEST_APP_URL }}"
```